ci/release/update-version.sh

```diff
@@ -87,6 +87,7 @@ DEPENDENCIES=(
 for DEP in "${DEPENDENCIES[@]}"; do
   for FILE in dependencies.yaml conda/environments/*.yaml; do
     sed_runner "/-.* ${DEP}==/ s/==.*/==${NEXT_SHORT_TAG_PEP440}.*/g" ${FILE}
+    sed_runner "/-.* ${DEP}-cu.*==/ s/==.*/==${NEXT_SHORT_TAG_PEP440}.*/g" ${FILE}
```

Comment:
This suggestion unifies the two sed commands. It works for me on Ubuntu. @raydouglass Do you also need to test this sed command on your Mac?
```suggestion
    sed_runner "/-.* ${DEP}\(-cu[0-9][0-9]\)\?/ s/==.*/==${NEXT_SHORT_TAG_PEP440}.*/g" ${FILE}
```